build.gradle

```diff
@@ -228,6 +228,32 @@ project('spring-rabbit') {
 
 }
 
+project('spring-rabbit-test') {
+	description = 'Spring Rabbit Test Support'
+
+	compileTestJava {
+		sourceCompatibility = 1.8
+		targetCompatibility = 1.8
+	}
+
+	dependencies {
+
+		compile project(":spring-rabbit")
+		compile ("junit:junit:$junitVersion") {
+			exclude group: 'org.hamcrest'
+		}
+		compile "org.hamcrest:hamcrest-all:$hamcrestVersion"
+		compile ("org.mockito:mockito-core:$mockitoVersion") {
+			exclude group: 'org.hamcrest'
+		}
+		testCompile project(":spring-rabbit").sourceSets.test.output
+		testCompile "log4j:log4j:$log4jVersion"
```

Comment:
This is redundant. See `// dependencies that are common across all java projects`.
But yes, it is minor, of course.

As you said you would like do not spend more time on this and push it to public and get some feedback to go ahead.
So, feel free to document it and I'll review and merge.